src/Microsoft.DotNet.Wpf/src/PresentationFramework/MS/Internal/AppModel/AssemblyFilter.cs

```diff
@@ -47,28 +47,29 @@ internal void FilterCallback(Object sender, AssemblyLoadEventArgs args)
                 // other wise also we cannot do any processing on a reflection only assembly aside from reflection based actions
                 if (!a.ReflectionOnly)
```

Comment:
@singhashish-wpf complete block inside "if" is commented out. Shall this "if" go away? even everything inside "lock" may not be needed, if following statement do not perform an action.
Assembly a = args.LoadedAssembly;

Reply:
Updaetd in commit https://github.com/dotnet/wpf/pull/4968/commits/1e55e8532fffbff1dcb526e91fbbca6dd324c5ab